code/controllers/subsystem/processing/flightpacks.dm

```diff
@@ -5,7 +5,7 @@ var/datum/subsystem/processing/flightpacks/SSflightpacks
 	priority = 5
 	wait = 2
 	stat_tag = "FM"
-	flags = SS_NO_INIT|SS_FIRE_IN_LOBBY|SS_KEEP_TIMING
+	flags = SS_NO_INIT|SS_TICKER|SS_KEEP_TIMING
```

Comment:
This was my initial intent but now that I look at it.. this miiiight be a baaad ideaaa..... hmmmmmmm...